grails-app/assets/javascripts/permissionTable.js

```diff
@@ -141,6 +147,11 @@ function initialise(roles, currentUserId, hubId, containerId) {
         });
     });
 
+    $('#email').on('change', function () {
```

Comment:
This should be in the SearchUserHubPermissionViewModel and attached via a binding

Reply:
my initial implementation was to include this in SearchUserHubPermissionViewModel to attach binding, however I cannot make the code below to work as the table will become "undefined" as it's needed to be inside the table initialisation.

table.search( val ).draw();

Reply:
The way you would fix that is to either:
1) Pass the table to the view model constructor.
2) Initialise the table itself in the view model and save the reference.

However, since a lot of the table code is already using jquery style event handlers and the view model appears to not be doing anything, you might as well just delete the view model as it appears to be unused in the current implementation?